common.py

```diff
@@ -76,6 +76,34 @@ class IperfOutput():
     command: str
     result: dict
 
+@dataclass
+class PluginOutput():
+    plugin_metadata: dict
+    command: str
+    result: dict
+    name: str
+
+@dataclass
+class TftAggregateOutput():
+    iperf_output: IperfOutput = None
+    measureCPU_client_output: PluginOutput = None
+    measureCPU_server_output: PluginOutput = None
+    measurePower_client_output: PluginOutput = None
+    measurePower_server_output: PluginOutput = None
```

Comment:
All these types should be wrapped in optional since they are not required.

```
from typing import Optional

...

    iperf_output: Optional[IperfOutput] = None